Minor updates to deploy overview (#15606)
PrefectHQ/prefect: docs/3.0/deploy/index.mdx

```

The `serve` method creates a deployment from your flow and immediately begins listening for scheduled runs to execute. 

> This example also creates a schedule so that the flow will run every minute of every day.

Reviewer: ```suggestion
Providing `cron="* * * * *"` to `.serve` associates a schedule with your flow so it will run every minute of every day.
```
